nixos/tests/switch-test.nix

```
@@ -68,9 +68,9 @@ in
             echo "systemd 0" > $out/init-interface-version
           '';
 
-          modifiedSystemConf.configuration.systemd.extraConfig = ''
-            # Hello world!
-          '';
+          modifiedSystemConf.configuration.systemd.settings.Manager = {
+            DummyTestConfig = "Hello World!";
+          };
```

This will start failing if we become able to test these configurations against the schema at build time.

```suggestion
          modifiedSystemConf.configuration.systemd.settings.Manager = {
            RuntimeWatchdogSec = "auto";
          };
```

or something? Or the other test does `DefaultEnvironment = "XXX_SYSTEM=foo";`.